### PR TITLE
Uplink discounts

### PR DESCRIPTION
**Pull request by whateverusername0**
## About the PR

round 2 of cooking
uplink discounts
ported & edited from [WWDP](https://github.com/WWhiteDreamProject/wwdpublic/pull/11)

## Why / Balance

gaming

## Media

image
the sheer luck

:cl:
- add: The Syndicate has been packing in items lately, so they've decided to start a yard sale for their agents!

### CODE REVIEW

**Comment by jorgun** [comment]
is this shared. as in the same discount for all traitors?
seems these are random selected and a general max min for eatch item.
some items shud have a custom min. like some of the usless can have 100% off. while some of the more common like emag can at most be 10% off as it is populare

**Comment by whateverusername0** [comment]
> is this shared. as in the same discount for all traitors? seems these are random selected and a general max min for eatch item. some items shud have a custom min. like some of the usless can have 100% off. while some of the more common like emag can at most be 10% off as it is populare

shouldn't be shared for all traitors

**Comment by jorgun** [comment]
i damenad a 100% of on Clown duffel! infact you shud pay me to pick it up

**Comment by whateverusername0** [comment]
> i damenad a 100% of on Clown duffel! infact you shud pay me to pick it up

when you code contractor kit :trollface:

**Comment by jorgun** [comment]
https://github.com/space-wizards/space-station-14/pull/26297
looks like there is an discount coming from upstream. 
can you explain the difference betwen there two solutions?

**Comment by whateverusername0** [comment]
> can you explain the difference betwen there two solutions?

more visually appealing. and more shitcode :trollface:
if the upstream gets merged it'd be nice with all the store refactors, but the PR was opened back in march so it's gonna take a lot of time i believe

File: Resources/Prototypes/Goobstation/Datasets/uplink_discount_fluff.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+- type: dataset
+  id: UplinkDiscountFluff
+  values:
+  - "No refunds, no questions asked."
+  - "May contain flammable elements, proceed with caution."
+  - "We've got too many of this stuff laying on our shelves."
```
Comment (Piras314) [inline Resources/Prototypes/Goobstation/Datasets/uplink_discount_fluff.yml:6]: grammar police "We've got too much of this laying around on our shelves."

**Review by Piras314 — CHANGES_REQUESTED** [review]

**Pull request merged**